src/dataexplorer/dataexplorer-application/src/components/PatientQuestionnaireResponses/PatientQuestionnaireResponses.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import useFetch from 'utils/hooks/useFetch';
+import { IQuestionnaireResponse } from 'types/IQuestionnaireResponse';
+
+const PatientQuestionnnaireResponses = (props: { patientID: string }) => {
+    const { response: patientQuestionnaireResponses } = useFetch<
+        IQuestionnaireResponse
+    >('fhir/QuestionnaireResponse?subject=Patient/' + props.patientID);
+    if (patientQuestionnaireResponses) {
+        return (
+            <h1>
+                Pasienten har
+                {patientQuestionnaireResponses?.entry?.length}
+                utfylte skjemaer
+            </h1>
+        );
+    }
+    return <></>;
+};
```

Comment:
I don't think this file is used anymore

Reply:
Yeah. I dont think the rebase recognized this, will remove.